test/libsolidity/syntaxTests/literalSuffixes/parameters/type_mismatch.sol

```diff
@@ -66,15 +66,15 @@ contract C {
     }
 }
 // ----
-// TypeError 8838: (635-647): The type of the literal cannot be converted to the parameter of the suffix function.
-// TypeError 8838: (657-672): The type of the literal cannot be converted to the parameter of the suffix function.
-// TypeError 8838: (721-735): The type of the literal cannot be converted to the parameter of the suffix function.
-// TypeError 8838: (745-758): The type of the literal cannot be converted to the parameter of the suffix function.
-// TypeError 8838: (808-823): The type of the literal cannot be converted to the parameter of the suffix function.
-// TypeError 8838: (833-848): The type of the literal cannot be converted to the parameter of the suffix function.
-// TypeError 8838: (858-876): The type of the literal cannot be converted to the parameter of the suffix function.
-// TypeError 8838: (925-942): The type of the literal cannot be converted to the parameter of the suffix function.
-// TypeError 8838: (952-968): The type of the literal cannot be converted to the parameter of the suffix function.
+// TypeError 8838: (635-647): The literal is out of range of type bool accepted by the suffix function.
+// TypeError 8838: (657-672): The literal is out of range of type address accepted by the suffix function.
+// TypeError 8838: (721-735): The literal is out of range of type string accepted by the suffix function.
+// TypeError 8838: (745-758): The literal is out of range of type bytes accepted by the suffix function.
+// TypeError 8838: (808-823): The literal is out of range of type int8 accepted by the suffix function.
+// TypeError 8838: (833-848): The literal is out of range of type bool accepted by the suffix function.
+// TypeError 8838: (858-876): The literal is out of range of type address accepted by the suffix function.
+// TypeError 8838: (925-942): The literal is out of range of type string accepted by the suffix function.
+// TypeError 8838: (952-968): The literal is out of range of type bytes accepted by the suffix function.
```

Comment:
This also sounds weird. "Out of range" only makes sense for integers. For other types we should just say that it cannot be converted.